package.json

```diff
@@ -16,6 +16,7 @@
     "debug": "^4.1.1",
     "ethers": "^4.0.47",
     "file-saver": "^2.0.2",
+    "lodash": "^4.17.20",
```

Comment:
Was this used anywhere?

Reply:
yup, i realise it is used in publish.tsx, but somehow not in the dependancies.